Assignment 1 - Matrices by Dr. Yamini Jangir.

SPA622M (Odd semester, 2024)

$(Q-1)$

2. Submit proof of any 10 highlighted theorems

(1) Prove that (AB)C = A(BC)

Let A,B and C be the matrix $a_{ij}, b_{ij}$ and $c_{ij}$. Then  multiplication of two matrices is given $$(AB)_{ij} = \sum_{k}(A_{ik}B_{kj})$$

Similarly we can write, 

$$((AB)C)_{rs} =  \sum_{w}(AB_{rw}C_{ws})$$

$$((AB)C)_{rs} =  \sum_{w}(\sum_{k}(A_{rk}B_{kw})C_{ws})$$

The left hand side can be written as, 

$$(A(BC))_{rs} =  \sum_{w}(A_{rw}(BC)_{ws})$$

$$(A(BC))_{rs} =  \sum_{w}\sum_{k}(A_{rw}(B_{wk}C_{ks}))$$

$$(A(BC))_{rs} =  \sum_{w}\sum_{k}(A_{rw}B_{wk})C_{ks})$$

$$(A(BC))_{rs} =  \sum_{w}\sum_{k}(AB)_{rk}C_{ks}) = (AB)C$$

$$ Hence Proved $$


(2) $(AB)^3 = B^\star A^\star$ 

We know that $$A^* = \bar{A}^T$$

$$ (AB)^* = \bar{AB}^T $$

$$ (AB)^* = (\bar{A}\bar{B})^T $$

$$ (AB)^* = (\bar{A}\bar{B})^T $$

On the basis of the transpose property, 

$$ (AB)^* = \bar{B}^T\bar{A}^T $$

$$ (AB)^* = B^*A^* $$



(3) All eigen values of a Hermitian Matrix are real numbers. 

Let the orthonormal basis be the $\ket{x_i}$ and we know that for any matrix A, we know

$$ A\ket{x_i} =  \lambda\ket{x_i}$$

Taking the Hermitian on the both sides, 

$$ \bra{x_i}A^* =  \lambda^*\bra{x_i}$$

$$ \braket{x_i|A^*|x_j} =  \lambda^*\braket{x_i|x_j}$$

Since A is a Hermitian Matrix, $A = A^*$ so, 

$$ \braket{x_i|A|x_j} =  \lambda^*\braket{x_i|x_j}$$

$$ \braket{x_i|\lambda|x_j} =  \lambda^*\braket{x_i|x_j}$$

$$ \braket{x_i|\lambda|x_j} =  \lambda^*\braket{x_i|x_j}$$

So as these basis are orthonormal basis, 

$$ \lambda\braket{x_i|x_j} =  \lambda^*\braket{x_i|x_j}$$

$$ \lambda =  \lambda^*$$

Hence Proved that the eigen values of the Hermitian Matrix are real.

(4) Prove that trac(ABC) = tra(CAB)

for the matrices A,B and C the elementary form can be written as 

$$A = a_{ij}$$

$$B = b_{ij}$$

$$C = c_{ij}$$

The multiplication of matrices A,B,C can be written as 

$$ ABC = \sum_{k}\sum_{r} a_{ik}b_{kj}c_{ji} $$

The trace of ABC is written as 

$$Tra(ABC) = (ABC)_{ii} = \sum_{i}\sum_{j}\sum_{k} a_{ik}b_{kj}c_{ji}$$

Using the property of the summation, 

$$ \sum_{i}\sum_{j}\sum_{k} a_{ik}b_{kj}c_{ji} = \sum_{i}\sum_{j}\sum_{k} b_{ik}c_{kj}a_{ji}  = tra(BCA)$$

Hence Proved


(5) All eigenvalues of a symmetric matrix are real numbers.

Let the orthonormal basis be the $\ket{x_i}$ and we know that for any matrix A, we know

$$ A\ket{x_i} =  \lambda\ket{x_i}$$

Given that A is the symmetric matrix, 

$$ A = A^T $$

Taking the complex conjugate on the both sides

$$ \bra{x_i}(A^T)^*  =  \lambda^*\bra{x_i}$$

Taking the inner product with $\ket{x_i}$

$$ \braket{x_i|A^*|x_j} =  \lambda^*\braket{x_i|x_j}$$

Assuming that matrix A has real entries, 

$$A = A^*$$

$$ \braket{x_i|A|x_j} =  \lambda^*\braket{x_i|x_j}$$

$$\lambda = \lambda^*$$

(6) Prove $(AB)^T = B^TA^T$

for the matrices A,B and C the elementary form can be written as 

$$A = a_{ij} , A^T = a_{ji}$$

$$B = b_{ij},  B_T = b_{ji} $$

$$AB =\sum_{k} a_{ik}b_{kj}$$

Taking Transpose on the both side, 

$$ (AB)^T = (\sum_{k} a_{ik}b_{kj})^T $$

$$ (AB)^T = (\sum_{k} b_{jk}a_{ki}) $$

Also, 

$$ B^TA^T= \sum_{k}b_{jk}a_{ki} $$

LHS = RHS...Hence Proved

(7) Prove that $ (AB) \neq (BA)$ 

For the matrices A,B the elementary form can be written as 

$$A = [a_{ij}]_{mn} $$

$$B = [b_{ij}]_{nm} $$

$AB = m * m$ matrix 

Also, 

$BA = n *n $ matrix 

Since the order of the matrices are different

$$ (AB) \neq (BA)$$

(8) Prove that det(A) = det(A^T)

we know that the determinant of a matrix  A is written as

$$det(A) = \sum_{k}a_{ij}(-1)^{i+j}M_{ij}$$

Taking the transpose of det(A)

$$det(A^T) = \sum_{k}a_{ji}(-1)^{j+i}M_{ji}$$

Swapping i to j and j to 1 

$$det(A^T) = \sum_{k}a_{ij}(-1)^{i+j}M_{ij}$$

Hence Proved that det(A) = det(A^T)


(9) Eigenvectors corresponding to distinct eigenvalues of a symmetric matrix are orthogonal.

Let A be the symmetric matrix where $ A = A^T $

Let $\ket{x_i}$ and $\ket{x_j}$ be the two different eigen basis and

$$A\ket{x_i} = \lambda_{1}\ket{x_i}$$

$$A\ket{x_j} = \lambda_{2}\ket{x_j}$$

We can write that

$$\braket{x_i|A|x_j} = \lambda_{1}\braket{x_i|x_j}$$ 

Assuming that entries of the matrix A is real using the transpose property, 

$$\braket{A x_i|x_j} = \lambda_{2}\braket{x_i|x_j}$$

Substracting the two equations, 

$$ (\lambda_1 - \lambda_2) \braket{x_i|x_j}  = 0 $$

Since eigen values are distinct 

$$\braket{x_i|x_j}  = 0$$

Hence Proved


(10) The determinant of an orthogonal matrix is either 1 or −1.

For the orthogonal matrix A 

$$ AA^T = A^TA = I$$

Taking the determinant on the both sides 

$$ det(AA^T) = det(A^TA) = det(I) $$

By the property of the determinent, 

$$ det(AA^T) = det(A)det(A^T) $$

We know that $det(A) = det(A^T)$

$$ (det(A))^2 = 1 $$

$$ det(A) = \pm 1$$

Hence Proved


($Q - 2$)   Find the inverse of the following matrix and verify by coding in python

(a)
$$
A = \left[\begin{array}{ccc}
3  & 2 & 1
\\
2 & 2 & 1
\\
1 & 1 & 4
\end{array}\right]
$$
The determinant of the matrix is det(A) = 3(8-1) - 2(8-1) + 1(2-2) = 21 -14 = 7

So the inverse of the matrix is can be written as 

$$A^{-1} = \frac{adj(A)}{det(A)}$$ 

The Adjoint of the matrix A is written as $$adj(A) = C_{ij} = (-1)^{i+j}M_{ij}$$

Calculating all the $C_{ij}s$

$C_{11} = M_{11} = 7$  , 
$C_{12} = -M_{12} = 7$ ,
$C_{13} = M_{13} = 0$,
$C_{21} = -M_{21} = 7$

$C_{22} = M_{22} = 13$,
$C_{23} = -M_{23} = 5$,
$C_{31} = M_{31} = 0$,
$C_{32} = -M_{32} = 5$,
$C_{33} = M_{33} = 7$

So the adjoint of the matrix A i.e. adj(A) is 

$$
adj(A) = \left[\begin{array}{ccc}
7  & 7 & 0
\\
7 & 13 & 5
\\
0 & 5 & 7
\end{array}\right]
$$ 

And the inverse of the matrix A is given by 

$$
A^{-1} = \frac{adj(A)}{det(A)} = \frac{1}{7}\left[\begin{array}{ccc}
7  & 7 & 0
\\
7 & 13 & 5
\\
0 & 5 & 7
\end{array}\right]
$$


(b)

$$
A = \left[\begin{array}{ccc}
cos(\theta)  & 0 & sin(\theta)
\\
0 & 1 & 0
\\
-sin(\theta)  & 0 & cos(\theta)
\end{array}\right]
$$
The determinant of the matrix is det(A) = $cos^2(\theta) + sin^2(\theta) = 1$ 

So the inverse of the matrix is can be written as 

$$A^{-1} = \frac{adj(A)}{det(A)}$$ 

The Adjoint of the matrix A is written as $$adj(A) = C_{ij} = (-1)^{i+j}M_{ij}$$

Calculating all the $C_{ij}s$

$C_{11} = M_{11} = cos(\theta)$  , 
$C_{12} = -M_{12} = 0$ ,
$C_{13} = M_{13} = -sin(\theta) $,
$C_{21} = -M_{21} = 0 $

$C_{22} = M_{22} = 1 $,
$C_{23} = -M_{23} = 0$,
$C_{31} = M_{31} = sin(\theta) $,
$C_{32} = -M_{32} = 0$,
$C_{33} = M_{33} = cos(\theta)$

So the adjoint of the matrix A i.e. adj(A) is 

$$
adj(A) = \left[\begin{array}{ccc}
cos(\theta)  & 0 & -sin(\theta)
\\
0 & 1 & 0
\\
sin(\theta) & 0 & cos(\theta)
\end{array}\right]
$$ 

And the inverse of the matrix A is given by 

$$
A^{-1} = \frac{adj(A)}{det(A)} = \left[\begin{array}{ccc}
cos(\theta)  & 0 & -sin(\theta)
\\
0 & 1 & 0
\\
sin(\theta) & 0 & cos(\theta)
\end{array}\right]
$$

In [1]:
##verification  (Q-2) A
import numpy as np 

A = np.array([[3,2,1],[2,2,1],[1,1,4]])

print(np.linalg.det(A))
print(np.linalg.inv(A))

7.000000000000001
[[ 1.         -1.          0.        ]
 [-1.          1.57142857 -0.14285714]
 [ 0.         -0.14285714  0.28571429]]


In [18]:
##verification  (Q-2) B
theta = np.pi/4

A = np.array([[np.cos(theta),0,np.sin(theta)],[0,1,0],[-np.sin(theta),0,np.cos(theta)]])

print(np.linalg.det(A))
print(np.linalg.inv(A))


1.0
[[ 0.70710678  0.         -0.70710678]
 [ 0.          1.          0.        ]
 [ 0.70710678  0.          0.70710678]]


$(Q-3)$ Find the solution of the following system of linear equation and verify by coding in python

$$ x_1 + 2x_2 = 3$$
$$ 2x_1 + 4x_2 = 6$$

Given are the linear non-homogeneous equations so using the Cramer's Method

$$ AX = B $$ 

where

$$
A = \left[\begin{array}{cc}
1 & 2 
\\
2 & 4
\end{array}\right] , B = \left[\begin{array}{c}
3  
\\
6 
\end{array}\right] , X = \left[\begin{array}{c}
x  
\\
y
\end{array}\right]  
$$

Calcuating the determinant of A i.e. det(A) = 4-4 = 0. Since the det(A) = 0 the given linear systems of equation has infinite solutions or indefinite solutions. 
Inorder to check it 

$$
D_x = \left[\begin{array}{cc}
3 & 2 
\\
6 & 4
\end{array}\right] 
$$ 

The determinant of the D_x is 0 i.e.  D_X = 0, similarly D_y = 0 
So, the given system of solutions has indefinite solutions 

In [4]:
##Python verification Q-3

A = np.array([[1,2],[2,4]])
B = np.array([3,6])

try:
    sol = np.linalg.solve(A,B)
except:
    print("Singular Matrix")

Singular Matrix


$(Q-4)$ Arfken Question 

One description of spin-1 particles uses the matrices

$$ M_x = \frac{1}{\sqrt2}\left[\begin{array}{ccc}
0 & 1 & 0          
\\
1 & 0 & 1
\\
0 & 1  & 0
\end{array}\right] , M_y = \frac{1}{\sqrt2}\left[\begin{array}{ccc}
0 & -i & 0          
\\
-i & 0 & -i
\\
0 & -i & 0
\end{array}\right], M_z = \frac{1}{\sqrt2}\left[\begin{array}{ccc}
1 & 0 & 0          
\\
0 & 0 & 0
\\
0 & 0 & -1
\end{array}\right]

$$

1. Prove $[M_x,M_y] = iM_Z$

$$
[M_x,M_y] = M_xM_y - M_yM_x  = \frac{1}{2}\left[\begin{array}{ccc}
0 & 1 & 0          
\\
1 & 0 & 1
\\
0 & 1  & 0
\end{array}\right] \left[\begin{array}{ccc}
0 & -i & 0          
\\
-i & 0 & -i
\\
0 & -i & 0
\end{array}\right] - \frac{1}{2}\left[\begin{array}{ccc}
0 & -i & 0          
\\
-i & 0 & -i
\\
0 & -i & 0
\end{array}\right]\left[\begin{array}{ccc}
0 & 1 & 0          
\\
1 & 0 & 1
\\
0 & 1  & 0
\end{array}\right] 
$$

$$
= \frac{1}{2}(\left[\begin{array}{ccc}
i & 0 & -i          
\\
0 & 0 & 0
\\
i & 0  & -i
\end{array}\right] - \left[\begin{array}{ccc}
-i & 0 & -i          
\\
0 & 0 & 0
\\
i & 0  & i
\end{array}\right]) = \left[\begin{array}{ccc}
-i & 0 & 0          
\\
0 & 0 & 0
\\
0 & 0  & i
\end{array}\right]  = i M_z
$$

2. $ M^2 = M_x^2 + M_y^2 + M_z^2  = 2I_3$

$$
M_x^2 = M_x*M_x = \frac{1}{2}  \left[\begin{array}{ccc}
1 & 0 & 1          
\\
0 & 2 & 0
\\
1 & 0 & 1 
\end{array}\right]
$$

$$
M_y^2 = M_y*M_y = \frac{1}{2}  \left[\begin{array}{ccc}
1 & 0 & -1          
\\
0 & 2 & 0
\\
-1 & 0 & 1 
\end{array}\right]
$$

$$

M_z^2 = M_z*M_z = \frac{1}{2}  \left[\begin{array}{ccc}
1 & 0 & 0          
\\
0 & 0 & 0
\\
0 & 0 & 1 
\end{array}\right]

$$

$$ 
M^2 =  \frac{1}{2}\left[\begin{array}{ccc}
1 & 0 & 1          
\\
0 & 2 & 0
\\
1 & 0 & 1 
\end{array}\right] +\left[\begin{array}{ccc}
1 & 0 & -1          
\\
0 & 2 & 0
\\
-1 & 0 & 1
\end{array}\right] + \left[\begin{array}{ccc}
 1 & 0 & 0          
\\
0 & 0 & 0
\\
0 & 0 & 1
\end{array}\right] = 2I_3
$$

3. $L_+ = M_x + iM_y$ and $L_− = M_x − iM_y$

[M^2, ]

$$
L_+ = \left[\begin{array}{ccc}
0 & \sqrt2 & 0          
\\
0 & 0 & \sqrt2
\\
0 & 0 & 1
\end{array}\right]

$$
$$
L_- = \left[\begin{array}{ccc}
0 & 0 & 0          
\\
\sqrt2 & 0 & 0
\\
0 & \sqrt2 & 1
\end{array}\right]
$$


(a) $M_2M_i −M_iM_2 = IM_i - M_iI = 0$

(b) $[Mz,L_x] = M_z L_x - L_x M_z = M_z(M_x + iM_y) - (M_x + iM_y)M_z= M_zM_x + iM_zM_y - M_zM_x + iM_yM_z  = −i[M_y,M_z] − [M_x,M_z] = M_x + M_yi = L_x$ 

(c) $[L_{+}, L_{−}] = L_xL_y - L_yL_x = (M_x + iM_y)(M_x - iM_y) - (M_x + iM_y)(M_x - iM_y) = M_x^2 + M_y^2 - (M_x^2 + M_y2) - i[M_x,M_y] - - i[M_x,M_y] = 2M_z  $
    $   $

$(Q-5)$  Diagonalize the following matrices and verify by coding in python

$$
A = \left[\begin{array}{cc}
5 & 1 
\\
1 & 5  
\end{array}\right]
$$


A matrix A is said to be diagonalizable if there exists a basis of eigenvectors for the vector space. This
means that the matrix can be represented in the form:

$$D = X^{−1}AX$$  

where D is a diagonal matrix whose diagonal elements are the eigenvalues of A, and X is a matrix
whose columns are the corresponding eigenvectors.  

For calculating the eigen values, using the Caliy-Hamiltonian Charateristic theorem

$$
det(A-\lambda I) = 0
$$

$$(5-\lambda)^2 - 1 = 0$$

$$ (\lambda^2 - 10\lambda + 24) = 0 $$

Solving these equations $$\lambda = 4,6$$, 

Calculaing the eigen vectors, 

$$
v1 = \left[\begin{array}{c}
-1 
\\
1   
\end{array}\right]  , v_2 = \left[\begin{array}{c}
1 
\\
1   
\end{array}\right]
$$

Then the matrix P is given by in terms of eigen vectors is 

$$
P = \left[\begin{array}{cc}
-1 & 1
\\
1  & 1   
\end{array}\right]
$$

Then the inverse of P is given by 

$$
P^{-1} = \frac{1}{2} \left[\begin{array}{cc}
-1 & 1
\\
1  & 1   
\end{array}\right]

$$



So 
$$

P^{-1}AP = \frac{1}{2} \left[\begin{array}{cc}
-1 & 1
\\
1  & 1   
\end{array}\right] \left[\begin{array}{cc}
5 & 1 
\\
1 & 5  
\end{array}\right] \left[\begin{array}{cc}
-1 & 1
\\
1  & 1   
\end{array}\right] = \left[\begin{array}{cc}
4 & 0
\\
0  & 6   
\end{array}\right]


$$


(b) 

$$ 
B = \left[\begin{array}{ccc}
1 & 0 & 1
\\
0  & 3 & 2
\\
0 & 0 & 2
\end{array}\right]

$$

A matrix A is said to be diagonalizable if there exists a basis of eigenvectors for the vector space. This means that the matrix can be represented in the form:

$$D = X^{−1}BX$$  

where D is a diagonal matrix whose diagonal elements are the eigenvalues of A, and X is a matrix
whose columns are the corresponding eigenvectors.  

For calculating the eigen values, using the Caliy-Hamiltonian Charateristic theorem

$$
det(B-\lambda I) = 0
$$

$$ 
B = \left[\begin{array}{ccc}
1-\lambda & 0 & 1
\\
0  & 3-\lambda & 2
\\
0 & 0 & 2 - \lambda
\end{array}\right]

$$

$$ det(A - \lambda I) = (1-\lambda)(3 - \lambda)(2 - \lambda) = 0 $$

$$ \lambda = 1,2,3 $$

The corresponding eigen vectors are 

$$
v1 = \left[\begin{array}{c}
1 
\\
0
\\
0   
\end{array}\right]  , v_2 = \left[\begin{array}{c}
1 
\\
-2
\\
1   
\end{array}\right] ,v_3 = \left[\begin{array}{c}
0
\\
1
\\
0   
\end{array}\right]
$$

Then the P matrix used for the diagonalization is

$$ 
P = \left[\begin{array}{ccc}
1 & 1 & 0
\\
0  & -2 & 1
\\
0 & 1 & 0
\end{array}\right]

$$

$$ det(B) = 1 $$

Then the 
$$
P^{-1} = \left[\begin{array}{ccc}
1 & 0 & -1
\\
0  & 0 & 1
\\
0 & 1 & 2
\end{array}\right]

$$
So,
$$

P^{-1}BP = \left[\begin{array}{ccc}
1 & 0 & -1
\\
0  & 0 & 1
\\
0 & 1 & 2
\end{array}\right] \left[\begin{array}{ccc}
1 & 0 & 1
\\
0  & 3 & 2
\\
0 & 0 & 2
\end{array}\right] \left[\begin{array}{ccc}
1 & 1 & 0
\\
0  & -2 & 1
\\
0 & 1 & 0
\end{array}\right] =  \left[\begin{array}{ccc}
1 & 0 & 0
\\
0  & 3 & 0
\\
0 & 0 & 2
\end{array}\right] 
$$









In [12]:
#(Q-5) Verification by python

import sympy

A = sympy.Matrix([[5,1],[1,5]])
B = sympy.Matrix([[1,0,1],[0,3,2],[0,0,2]]) 

A_dia = A.diagonalize()
B_dia = B.diagonalize()
print(A_dia)
print(B_dia)

(Matrix([
[-1, 1],
[ 1, 1]]), Matrix([
[4, 0],
[0, 6]]))
(Matrix([
[1,  1, 0],
[0, -2, 1],
[0,  1, 0]]), Matrix([
[1, 0, 0],
[0, 2, 0],
[0, 0, 3]]))


$(Q-6)$ A satellite captures images that are distorted due to the motion of the spacecraft. The distortion can be corrected using an affine transformation matrix T.

1.  Affine transformation matrix is used correct the distorted images. It is associated with the translation of axes, rotation and scaling. If the pixel is (x,y) then then T matrix can be written as 

$$
\left[\begin{array}{c}
x^1  
\\
y^1
\end{array}\right] = \left[\begin{array}{cc}
a & b
\\
c & d
\end{array}\right] \left[\begin{array}{c}
x  
\\
y
\end{array}\right]  
$$

where $x^1$ and $y^1$ are the coordinates of the pixel. 


2. Using the NumPy for the given affine matrix and the pixel coordinates 

$$
T = \left[\begin{array}{ccc}
1 & 0.1 & 0 
\\
0.1 & 1 & 0
\\
0 & 0 & 1
\end{array}\right]
$$

$$
P1 = \left[\begin{array}{c}
100 
\\
200 
\\
1
\end{array}\right] , P2 = \left[\begin{array}{c}
300 
\\
400 
\\
1
\end{array}\right]
$$

In [ ]:
## (Q-6) 

P1 = np.array([100,200,1])
P2 = np.array([300,400,1])
T = np.array([[1,0.1,0],[0.1,1,0],[0,0,1]])

P1_new  = T@P1
P2_new  = T@P2

print(P1_new)
print(P2_new)


[120. 210.   1.]
[340. 430.   1.]


($Q-7$) In the study of spacecraft dynamics, it is crucial to understand the behavior of small oscillations around an equilibrium point. Consider a spacecraft with a symmetric mass distribution, operating in a two-dimensional plane. The dynamics of the system near the equilibrium point can be modeled using the following linearized equations of motion:


$$
\left[\begin{array}{c}
\ddot{x}  
\\
\ddot{y} 
\end{array}\right]=A \left[\begin{array}{c}
x  
\\
 y  
\end{array}\right]
$$

where A is a 2 × 2 matrix that encapsulates the effects of the spacecraft’s mass distribution and external
forces.
Given the matrix
$$
A = \left[\begin{array}{cc}
-4  & 2
\\
2 & 3
\end{array}\right]
$$


(1) Find the eigenvalues and eigenvectors of the matrix A.

For calculating the eigen values, using the Caliy-Hamiltonian Charateristic theorem

$$
det(A-\lambda I) = 0
$$

$$
A - \lambda I = \left[\begin{array}{cc}
-4 -\lambda  & 2
\\
2 & 3 - \lambda
\end{array}\right]
$$

$$
det(A-\lambda I) = (-4-\lambda)(-3-\lambda) - 4
$$

Thereby solving the quadratic equation 

$$
\lambda^2 + 7\lambda +8 = 0 
$$

which gives the value of $\lambda$ as
$$\lambda = \frac{-7\pm\sqrt17}{2}$$

Using the NumPy module

In [3]:
##Solving using Numpy
import numpy as np

A = np.array([[-4,2],[2,-3]])

eig_values,eig_vectors = np.linalg.eig(A)

print(eig_values)
print(eig_vectors)


[-5.56155281 -1.43844719]
[[-0.78820544 -0.61541221]
 [ 0.61541221 -0.78820544]]


On the basis of the concept that AX= $\lambda$ X, 

$$
\left[\begin{array}{c}
\ddot{x}  
\\
\ddot{y} 
\end{array}\right]=A \left[\begin{array}{c}
x  
\\
 y  
\end{array}\right] = \lambda \left[\begin{array}{c}
x  
\\
 y  
\end{array}\right]  
$$

Therefore,

$$
\left[\begin{array}{c}
\ddot{x}  
\\
\ddot{y} 
\end{array}\right]=\left[\begin{array}{c}
\lambda_1 x  
\\
\lambda_2 y  
\end{array}\right]

$$

Multiplying both sides by m i.e. the mass of the satellite, 


$$
\left[\begin{array}{c}
m\ddot{x}  
\\
m\ddot{y} 
\end{array}\right]=\left[\begin{array}{c}
\lambda_1 mx  
\\
\lambda_2 my  
\end{array}\right]

$$

$$
\left[\begin{array}{c}
F_x  
\\
F_y 
\end{array}\right]=\left[\begin{array}{c}
\lambda_1 my
\\
\lambda_2 my  
\end{array}\right] = \left[\begin{array}{c}
F_x  
\\
F_y 
\end{array}\right]=\left[\begin{array}{c}
-5.56155281 my
\\
-1.43844719 my  
\end{array}\right] 

$$

Since the both forces in the x-direction and y-direction has negative on the RHS we can safely say that these $F_x$ and $F_y$ are the restoring forces so the stability of the satellite is in the stable equilibrium i.e. satellite will restore to its original position if the external forces are applied on it. 

3) From the matrix  

$$
\left[\begin{array}{c}
F_x  
\\
F_y 
\end{array}\right] = \left[\begin{array}{c}
-5.56155281 my
\\
-1.43844719 my  
\end{array}\right] 

$$ 

$F_x = \lambda_1 mx$ and $F_y = \lambda_2 my$ so solving the Differential equation for the x and y direction the solution are 

$$ x(t) = c_1e^{λ_1t}v1 + c_2e^{λ2t}v2 $$

According to the given initial condition, $C_1$ = 1 and $C_2$ = 1

So, $$x(t) = e^{-5.56155281*t}v1 + e^{-5.56155281*t}v2$$ where v1, v2 are the eigen vectors 


3) Eigen vectors are the orthonomal basis of the motion of the space craft which governs the dynamical motion of the space craft around the earth